src/services/authentication.js

```diff
@@ -1683,6 +1689,40 @@ function getSSOLoginUrlClient ({ client, authorizationParams, ssoConnection }) {
   return ssoLoginUrl
 }
 
+function getTokenTypeTransformationsOrder (provider) {
```

Comment:
This name is not accurate. It can make you think all transformations happen in order. Maybe `getTokenTypeTransformationsToTry`.